Makefile

```diff
@@ -52,7 +52,7 @@ endif
 LDFLAGS= $(COMMON_FLAGS) \
 -Wall -Wl,--cref -Wl,--check-sections -Wl,--gc-sections -Wl,--unresolved-symbols=report-all -Wl,--warn-common \
 -Wl,--warn-section-align \
--save-temps -nostartfiles \
```

Comment:
@mmoskal the flag -nostartfiles must be removed in Makefiles and I have compile ok

Reply:
Remove it here, and below add this:

```
ifneq ($(CHIP_FAMILY), saml21)
LDFLAGS += -nostartfiles
endif
```